app/Http/Controllers/AdminDash.php

```diff
@@ -672,7 +672,7 @@ public function removeVisitor($id) {
                 ],
                 'http_errors' => false
             ];
-            $res = $client->request('DELETE', 'https://api.vatusa.net/v2/facility/'.Config::get('vatusa.facility').'/roster/manageVisitor/'.$id.'?apikey='.Config::get('vatusa.api_key'), $req_params);
+            $res = $client->request('DELETE', Config::get('vatusa.base').'/v2/facility/'.Config::get('vatusa.facility').'/roster/manageVisitor/'.$id.'?apikey='.Config::get('vatusa.api_key'), $req_params);
```

Comment:
See comment above - move to separate issue.

Reply:
Removed